dts/bindings/sensor/asair,aht20.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023, Konrad Sikora
+# SPDX-License-Identifier: Apache-2.0
+
+description: Asair AHT20 Integrated temperature and humidity sensor
+
+compatible: "asair,aht20"
```

Comment:
`asair` vendor needs to be added to `vendor-prefixes.txt`

Reply:
thanks, I think I should go back to vendor "aocang" instead of "asair" because one sensor on the repository already exists with this vendor name (aosang dht).
It looks like asair and aosang are the same vendor http://www.aosong.com/en/products-32.html